### DISCUSSION TITLE
@typeschema/zod issue on Vercel

### DISCUSSION THREAD

**Discussion by Kavehrafie** [next-safe-action/next-safe-action#168] [General]
The deployment works fine but using the form with the package generate an error in the action: 
Cannot find package '@typeschema/zod' imported from /var/task/.next/server/chunks

**Comment by TheEdoRan** [comment]
How did you solve it?

**Reply by Kavehrafie** [reply]
I should have been more careful when upgrading to version 7.0. TypeSchema causes a crash on Edge while running locally, which is fine.
Since I am only using Zod and I don't need dynamic imports switched back to `import { createSafeActionClient } from "next-safe-action";`
Thanks!

**Reply by samie105** [reply]
I have been trying to solve this issue for 2 weeks now, but this works thanks a lot. it was so bad that I even had to contact vercel support
